### PR TITLE
1.13 Enchantments

### PR DESCRIPTION
**Pull request by ShaneBeee**
Target Minecraft versions: 1.13.x
Requirements: none
Related issues: #1385 

Description:
New 1.13 enchantments. 
Not much else to describe here ;)

### CODE REVIEW

File: src/main/resources/lang/english.lang [hunk 1]
```diff
@@ -238,6 +238,11 @@ enchantments:
 		# It's sweeping on 1.10 and below for some reason.
 		sweeping: Sweeping Edge
 		sweeping_edge: Sweeping Edge
+		# New 1.13 Enchantments
+		channeling: Channeling
```
Comment (Blueyescat) [inline src/main/resources/lang/english.lang]: channeling: Channeling, Channelling

Supporting British English ❤️ (My native language isn't even English)

Comment (ShaneBeee) [reply src/main/resources/lang/english.lang]: Ill quickly whip that change up... I just went with the actual name of the enchantment, but I see what you mean there!

Comment (ShaneBeee) [reply src/main/resources/lang/english.lang]: Changes Complete

**Comment by bensku** [comment]
Appears to work.

**Pull request merged**